File: Bab 4/Exercise/4. Reamur.c
/*
 *  Name Program    : Reamur
 *  Purpose         : Try Code from Logic Algorithm and Basic Programming Book chapter 3
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : C Language
 */

#include <stdio.h>

// Main function
int main(){

    // Variable instantiation
    int celcius;
    int reamur;

    printf("Masukan derajat celcius\n");

    // Get input from console
    scanf("%d", &celcius);

    // Calculate from celcius to reamur
    reamur = (4 * celcius) / 5;

    // Print to console
    printf("%d", reamur);
    
    return 0;
}

File: Chapter 4/Exercise/4. Reamur.java
/*
 *  Name Program    : Reamur
 *  Purpose         : Answer question from exercise Chapter 4 from Logic Algorithm and Basic Programming Book
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 15th April 2021
 *  Language        : Java Language
 */

import java.util.Scanner;

public class Main{

    // Main function
    public static void main(String[] args){

        // Variable instantiation
        int celcius = 0;
        int reamur = 0;

        Scanner sc = new Scanner(System.in);

        System.out.println("Masukan derajat celcius");

        // Get input from console. Example = 5
        try{
            
            celcius = sc.nextInt();
        
        }catch(Exception e){

        }

        // Calculate from celcius to reamur
        reamur = (4 * celcius) / 5;
        
        // Print to console
        System.out.println(reamur);

    }
}